examples/parlai_chat_task_demo/custom_input_and_messages/main.js

```diff
@@ -93,7 +93,7 @@ function RenderChatMessage({
         }
         message={message.text}
         taskData={message.task_data}
-        messageId={message.message_id}
+        messageId={message.update_id}
```

Comment:
Not sure if we want to change the `messageId` here, it's probably fine as it is maybe as that's more of `bootstrap-chat`'s data model?

Reply:
Exactly my thought - `messageId` is more what you expect for what `bootstrap-chat` considers the updates to be.